life/modernized/18-chapter-18.adoc

```diff
@@ -458,8 +465,13 @@ So come, dear Lord Jesus, who was dead, and is alive, and lives forever.
 Amen.
 Very dearly farewell.
 
+[.signed-section-closing]
 Your friend, that faithfully travails for your redemption,
```

Comment:
This is a real borderline [sscl]!!! I will change it if you want, both ways looked good to me. 😀